docs/NMOS-With-IPMX-HKEP.md

```diff
@@ -46,12 +46,16 @@ A Sender MAY provide a `urn:x-nmos:cap:transport:hkep` capability to indicate th
 
 > Note: HKEP is only available for RTP based transport protocols, all of which use an SDP transport file.
 
+An example Sender and Receiver resources are provided in [Examples](./examples).
+
 ### SDP Transport File
 
 The `hkep` attribute defined in [TR-10-5][] is not yet registered with IANA. If it were, the definition would indicate "Usage Level: session, media" meaning that a session-level `hkep` attribute represents the default value for a media-level `hkep` attribute if the latter is not specified. The SDP transport file may therefore provide `hkep` information either at the session level and/or the media level.
```

Comment:
```suggestion
The `hkep` attribute defined in [TR-10-5][] is used in the SDP transport files to indicate that HDCP encryption and the HKEP protocol are used.
```

Reply:
I made a slight change "the SDP transport files" to "an SDP transport file". I hope it is ok for you.